entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
I think we need to decide when and how we want additional using statements to show up. E.g.
- This particular using statement is required pretty much any time you use EF Core APIs. We could choose to show it only in introductory material, mention it in every page that contains code snippets or sometimes (i.e. inconsistently) 
- Using statements are code. It could be better to have them incorporated in the code snippet rather than as a note.
- Some using statements are required by multiple snippets in this page. If we are going to have notes, is it better to have it at the top or at the bottom. 
- Notes really should be quoted and start with a [!NOTE] header

@Rick-Anderson @tdykstra how are we dealing with this in other parts of the documentation?